src/ui/shared/Reply.tsx

```diff
@@ -10,14 +10,28 @@ import {
 import * as React from 'react';
 
 import ErrorToast from 'fontoxml-feedback/src/ErrorToast';
-import { BusyState } from 'fontoxml-feedback/src/types';
+import {
+	BusyState,
+	CardContentComponentProps
+} from 'fontoxml-feedback/src/types';
 import t from 'fontoxml-localization/src/t';
 
 import AuthorAndTimestampLabel from '../AuthorAndTimestampLabel';
 import { CARD_HEADER_HEIGHT } from './../constants';
 import ReplyActionsDrop from './ReplyActionsDrop';
 import TruncatedText from './TruncatedText';
 
+type Props = {
+	reviewAnnotation: CardContentComponentProps['reviewAnnotation']; 
+	onCancelRetryRemove: CardContentComponentProps['onReplyFormCancel'];
+	onHide: CardContentComponentProps['onReplyErrorHide'];
+	onRefresh: CardContentComponentProps['onReplyRefresh'];
+	onRemove: CardContentComponentProps['onReplyRemove'];
+	onShowEditForm: CardContentComponentProps['onReplyEdit'];
+	reply: ReplyType;
```

Comment:
`ReplyType` is unknown, you have to import it. But, I think that the right type is `Reply`.

Reply:
Yes, you're right.